src/components/tabset/tabset.component.spec.ts

```diff
@@ -116,12 +117,35 @@ describe('Tabset Component', () => {
         expect(component.onTabDeselect).toHaveBeenCalled();
     });
 
+    it('should change to new tab when tab is programmatically selected', () => {
+
+        const tab1 = getTabForClass(0, nativeElement);
+        const tab2 = getTabForClass(2, nativeElement);
+
+        // check first tab is active
+        expect(tab1.classList.contains('active')).toBeTruthy();
+
+        const tabButton: HTMLElement = nativeElement.querySelector('.button-primary');
+        expect(tabButton).toBeTruthy();
+
+        // programatically change tab
+        tabButton.click();
+
+        // check second tab is now active/first tab is not active
+        expect(tab2.classList.contains('active')).toBeTruthy();
+        expect(tab1.classList.contains('active')).toBeFalsy();
```

Comment:
It would be good to check that the tab content has activated as well. We have a ticket in the backlog about tabs activating but showing the wrong content.

Reply:
Could you look into this please?